fdbserver/workloads/ConflictRange.actor.cpp

```diff
@@ -75,7 +75,6 @@ struct ConflictRangeWorkload : TestWorkload {
 
 	ACTOR Future<Void> conflictRangeClient(Database cx, ConflictRangeWorkload *self) {
 		state int i;
-		state int j;
```

Comment:
Declaring these single letter state variables is definitely a bad practice, and we can see in this function that we are shadowing the state variable `i` multiple times. If you felt so inclined, I think the only place that the state version of `i` is being used is at line 185 (or 184 in your new version, I guess), and I don't think it actually needs to be state there. You could remove `state int i` as well and just use `int i` in the loop below.

Reply:
Done